charts/karmada/values.yaml

```diff
@@ -19,6 +19,9 @@ global:
 ## for more details about yaml anchors and aliases.
 karmadaImageVersion: &karmadaImageVersion latest
 
+podDisruptionBudget: &podDisruptionBudget {}
+  # maxUnavailable: 20%
```

Comment:
Does this mean all components will share the same budget?

Reply:
Each resource has its own `podDisruptionBudget` configuration which can get an override. This was following the existing pattern of `karmadaImageVersion` on line 20 but happy to remove the anchor tag if you suggest though.

Reply:
Looks good to me.